src/mbgl/renderer/renderer_impl.cpp

```diff
@@ -67,9 +67,14 @@ void Renderer::Impl::setObserver(RendererObserver* observer_) {
 }
 
 void Renderer::Impl::render(const UpdateParameters& updateParameters) {
-    // Don't load/render anyting in still mode until explicitly requested.
-    if (updateParameters.mode == MapMode::Still && !updateParameters.stillImageRequest) return;
-    
+    // Avoid redundant render calls in still mode if:
```

Comment:
Isn't this check redundant now that it's added above to `Map::Impl::onUpdate()`? If the `UpdateParameters` are not set, `Renderer::Impl::render` will not be called